dali/dfu/dfurun.cpp

```diff
@@ -1314,6 +1314,16 @@ class CDFUengine: public CInterface, implements IDFUengine
                                 };
                             }
                         }
+#ifdef _CONTAINERIZED
+                        StringBuffer clusterName;
+                        destination->getGroupName(0, clusterName);
+                        Owned<IPropertyTree> plane = getDropZonePlane(clusterName);
+                        if (plane)
+                        {
+                            unsigned np = plane->getPropInt("@defaultSprayParts", 0);
```

Comment:
minor: If not set, then 0, which is default internally for numpartsoverride, which means unset.

Would perhaps be clearer intention, if didn't call if not set, e.g.:
```
    if (plane->hasProp("@defaultSprayParts")
        destination->setNumPartsOverride(plane->getPropInt("@defaultSprayParts"));
```